src/main/java/org/apache/ibatis/mapping/CacheBuilder.java

```diff
@@ -37,7 +37,7 @@
 /**
  * @author Clinton Begin
  */
-public class CacheBuilder {
+public final class CacheBuilder {
```

Comment:
I don't think this class should be `final`,because there are some Third party libs' code could be extends `cacheBuilder`, like `RedisCacheBuilder`

Reply:
maybe the comments is too little...